src/System.Text.RegularExpressions/src/System/Text/RegularExpressions/Regex.cs

```diff
@@ -188,7 +188,7 @@ protected IDictionary Caps
         {
             get
             {
-                return caps;
+                return caps!;
```

Comment:
`caps` will be initialized unless Regex is inherited using the protected parameterless constructor. Not sure if we should declare this property nullable because of that.

Reply:
Sounds like nullable is better.